pkgdb/tests/buildenv.bats

```diff
@@ -1,273 +0,0 @@
-#! /usr/bin/env bats
-# --------------------------------------------------------------------------- #
-#
-# @file tests/buildenv.bats
-#
-# @brief Test building environments from lockfiles.
-#
-# Relies on lockfiles generated by `pkgdb` from flox manifests.
-#
-# These tests only check the build segment,
-# they do not check the resolution of manifests,
-# nor the activation of the resulting environments.
-# Such tests are found in `pkgdb` and `flox` respectively.
-#
-#
-# --------------------------------------------------------------------------- #
-#
-# TODO: Allow a path to a file to be passed.
-#
-#
-# --------------------------------------------------------------------------- #
-
-# bats file_tags=build-env
-
-load setup_suite.bash
-
-# --------------------------------------------------------------------------- #
-
-setup_file() {
-  : "${CAT:=cat}"
-  : "${TEST:=test}"
-  : "${MKDIR:=mkdir}"
-  export CAT TEST MKDIR
-  export LOCKFILES="$BATS_TEST_DIRNAME/data/buildenv/lockfiles"
-
-  # Always use a consistent `nixpkgs' input.
-  export _PKGDB_GA_REGISTRY_REF_OR_REV="${NIXPKGS_REV?}"
-}
-
-# ---------------------------------------------------------------------------- #
-
-# bats test_tags=single,smoke
-@test "Simple environment builds successfully" {
-  run "$PKGDB_BIN" buildenv "$LOCKFILES/single-package/manifest.lock"
-  assert_success
-}
-
-# bats test_tags=single,smoke
-@test "Inline JSON builds successfully" {
-  run "$PKGDB_BIN" buildenv "$(< "$LOCKFILES/single-package/manifest.lock")"
-  assert_success
-}
-
-# ---------------------------------------------------------------------------- #
-
-
-# ---------------------------------------------------------------------------- #
-
-# bats test_tags=single,binaries
-@test "Built environment contains binaries for v0 lock" {
-  run "$PKGDB_BIN" buildenv \
-    "$LOCKFILES/single-package/manifest.lock"
-  assert_success
-  store_path=$(echo "$output" | jq -er '.store_path')
-  assert "$TEST" -x "${store_path}/bin/vim"
-}
-
-
-# ---------------------------------------------------------------------------- #
-
-# bats test_tags=single,binaries
-@test "Built environment contains binaries for v1 catalog package" {
-  run --separate-stderr "$PKGDB_BIN" buildenv \
-    "$GENERATED_DATA/envs/hello/manifest.lock"
-  assert_success
-  store_path=$(echo "$output" | jq -er '.store_path')
-  assert "$TEST" -x "${store_path}/bin/hello"
-}
-
-
-# ---------------------------------------------------------------------------- #
-
-# bats test_tags=single,binaries
-@test "Built environment contains binaries for v1 flake package" {
```

Comment:
nonblocking: do we have coverage for flake packages somewhere?

Reply:
we're testing flake builds in `buildenv.rs`, the buildenv tests were also previously translated to rust and are now also in `buildenv.rs`